docs/docs/reference/changed-features/numeric-literals.md

```diff
@@ -112,6 +112,36 @@ class NumberTooSmall (msg: String = "number too small")         extends FromDigi
 class MalformedNumber(msg: String = "malformed number literal") extends FromDigitsException(msg)
 ```
 
+### Compiler Expansion
+
+The companion object `FromDigits` also defines four methods that may be used to provide a given instance of one of the subclasses of `FromDigits`:
```

Comment:
I believe this is not enough. For instance, it is said that the companion object defines a method `fromDecimalDigits` but it is not specified where and when this method is called. 

I believe if we want to this to go ahead, it needs a complete rework of the doc page, and the SIP committee has to decide on the changes. My concern is that once the additions are fully worked in, we might decide that things became too complex.